tests/acceptance/servicecatalog/servicecatalog_test.go

```diff
@@ -56,11 +63,16 @@ func TestServiceCatalogContainsRemoteEnvironmentBrokerServiceClass(t *testing.T)
 
 		deleteClusterServiceClassesForRemoteEnvironment(t, re)
 	}()
-	brokerServices = getServiceBrokerServices(t, os.Getenv(remoteEnvironmentBrokerURLEnvName))
 
 	// then
-	assert.NotEmpty(t, brokerServices)
-	// timeout must be greater than the broker relistDuration
+	wait.ForFuncAtMost(t, func() error {
```

Comment:
this test is a little bit different than for helm broker, here we create new RE and we should not only check length but if the list contains newly created RE.
Additionally, I would recommend renaming method: getServiceBrokerServices to getCatalogForBroker